README.md

```diff
@@ -2276,6 +2313,81 @@ The `rippled_server_status` object may have any of the following fields:
 
 # UTILITIES #
 
+## Sign Transaction ##
+[[Source]<br>](https://github.com/ripple/ripple-rest/blob/develop/api/transaction/sign.js#L50 "Source")
+
+Signs a Ripple transaction. The signed transaction can subsequently be submitted to the Ripple network with the [`submit`](#submit-transaction) endpoint. The transaction `hash` must be persisted before submitting so that the transaction can be verified.
+
+<!-- <div class='multicode'> -->
+
+<!-- *REST* -->
+
+```
+POST /v1/transaction/sign
+```
+
+<!-- </div> -->
+
+[Try it! >](https://ripple.com/build/rest-tool#sign-transaction)
+
+The following URL parameters are required by this API endpoint:
+
+| Field | Value | Description |
+|-------|-------|-------------|
+| tx_json | Object | A transaction object as returned by any of the [transaction endpoints](#transaction-endpoints) with the `?submit=false` query parameter |
+| secret | String | The Ripple secret for the account that is creating the transaction |
+
+#### Response ####
+
+The result is a JSON object, with a field `tx_blob` containing a signed transaction, and a field `hash` containing the transaction's hash.
+
+```js
+{
+  success: true,
+  tx_blob: '12000322000000002400000B7A201B0086955468400000000000000C732102F89EAEC7667B30F33D0687BBA86C3FE2A08CCA40A9186C5BDE2DAA6FA97A37D87446304402207660BDEF67105CE1EBA9AD35DC7156BAB43FF1D47633199EE257D70B6B9AAFBF022022A492002DC25C1C735458127FE195A24F362148536A11E48C04DBA515023B20770A726970706C652E636F6D81144FBFF73DA4ECF9B701940F27341FA8020C313443',
+  hash: '69FFB0855771C08E611F1CF79211C03494D20718F1F90A8FC1E265D2ACAD2CA9'
+}
+```
+
+## Submit Transaction ##
+[[Source]<br>](https://github.com/ripple/ripple-rest/blob/develop/api/transaction/submit.js#L5 "Source")
+
+Submits a signed transaction to the Ripple Network.
+
+<!-- <div class='multicode'> -->
+
+<!-- *REST* -->
+
+```
+POST /v1/transaction/submit
+```
+
+<!-- </div> -->
+
+[Try it! >](https://ripple.com/build/rest-tool#submit-transaction)
+
+The following URL parameters are required by this API endpoint:
+
+| Field | Value | Description |
+|-------|-------|-------------|
+| tx_blob | String | A signed transaction as returned by any of the [transaction endpoints](#transaction-endpoints) with the `?submit=false` query parameter or the [`sign`](#sign-transaction) endpoint |
+
+#### Response ####
+
+The result is a JSON object, with a field `engine_result` containing the result of the submission. A successful submission means that the transaction is pending validation, it is not valid until it has been accepted into a validated ledger.
+
+```js
+{
+  success: true,
```

Comment:
This is not valid JSON. Please use actual JSON that validates on jsonlint.com or some similar system.